From 4ececf7d42f7b97698cf1b3317ec849074551f4f Mon Sep 17 00:00:00 2001
From: Arun T <arun.t@intel.com>
Date: Sat, 4 Jul 2026 22:48:55 +0530
Subject: [PATCH 03182/15333] media: i2c: cvs: Add NVL ACPI ID

Add INTC10FA to the CVS ACPI match table so the driver can bind to
CVS devices exposed on NVL platforms.

Signed-off-by: Arun T <arun.t@intel.com>
Signed-off-by: Sakari Ailus <sakari.ailus@linux.intel.com>
---
 drivers/media/i2c/cvs/core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/cvs/core.c b/drivers/media/i2c/cvs/core.c
index 8270d93f1abe..7486815b16bd 100644
--- a/drivers/media/i2c/cvs/core.c
+++ b/drivers/media/i2c/cvs/core.c
@@ -965,6 +965,7 @@ static const struct acpi_device_id intel_cvs_acpi_match[] = {
 	{ "INTC10DE" }, /* LNL */
 	{ "INTC10E0" }, /* ARL */
 	{ "INTC10E1" }, /* PTL */
+	{ "INTC10FA" }, /* NVL */
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, intel_cvs_acpi_match);
-- 
2.55.0

